tf-build-tools/tf-maven-plugin/src/test/java/com/deliveredtechnologies/maven/terraform/TerraformPackageTest.java

```diff
@@ -165,7 +176,7 @@ public void packageGoalWithNoFatZipAndMultipleModulesPackagesBothTfSourceModules
         zipEntryNames.add(entry.getName());
       }
 
-      Assert.assertEquals(6, count);
+      Assert.assertEquals(4, count);
       Assert.assertTrue(zipEntryNames.contains("root/"));
```

Comment:
Assetions on strings produce more readable output on failure when using this style Assert.assertThat(tfCommand, CoreMatchers.containsString("-no-color "));